addons/website/views/snippets/s_features_grid.xml

```diff
@@ -2,62 +2,68 @@
 <odoo>
 
 <template id="s_features_grid" name="Features Grid">
-    <section class="s_features_grid pt48 pb24">
+    <section class="s_features_grid pt64 pb64" data-vcss="001">
         <div class="container">
             <div class="row">
                 <div class="col-lg-6 s_col_no_bgcolor pb24">
                     <div class="row">
-                        <div class="col-lg-12 pb24" data-name="Box">
-                            <h2>First list of Features</h2>
-                            <h5>Add a great slogan.</h5>
+                        <div class="col-lg-12" data-name="Box">
+                            <h2 class="h3-fs">Core Features</h2>
+                            <h5 class="lead">Essential tools for your success.</h5>
+                            <div class="s_hr pt32 pb-2">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
                         </div>
-                        <div class="col-lg-12 pt16 pb16" data-name="Box">
-                            <i class="fa fa-2x fa-font-awesome rounded-circle bg-o-color-1 s_features_grid_icon"/>
-                            <div class="s_features_grid_content">
-                                <h4>Change Icons</h4>
+                        <div class="col-lg-12 py-3" data-name="Box">
+                            <i class="fa fa-flag-o rounded bg-o-color-3 s_features_grid_icon float-start flex-shrink-0" role="img"/>
+                            <div class="s_features_grid_content mb-0 overflow-hidden">
+                                <h4 class="h5-fs">Change Icons</h4>
```

Comment:
Any reason why this could not be `<h3 class="h5-fs">` or just `<h5>`? I'm not sure I understand the goal of those `h{X}-fs` if it is not to respect the order of titles, but I might be missing something! 😄

Reply:
IIRC QSM requested to not change the main title of the snippet in order to ensure a right order of the headings of the page (e.g avoiding having a main snippet using a h4).

It's possible that this rule wasn't meant to be spread across the whole typescale but it looks like it has been 😅

Still, not a big deal IMO, as we keep the order of titles bug only tweak their rendering on the page.